### PR TITLE
Feat/subscopes

### PR DESCRIPTION
**Pull request by clairekinde11**
Added new section to the scopes doc for Kinde API and own API, to include info on:
- adding selected subscopes to the body of an access token request
- included a screen shot from postman

## Summary by CodeRabbit

- **New Features**
	- Added a section on requesting a subset of scopes for enhanced security in access token requests.
	- Introduced detailed examples for including scopes in token requests using Postman.
- **Documentation**
	- Updated existing documentation for clarity on scope management and token requests, including renaming sections and streamlining guidance.

### CODE REVIEW

File: src/content/docs/developer-tools/kinde-api/about-m2m-scopes.mdx [hunk 1]
```diff
@@ -14,22 +14,39 @@ app_context:
...
 - `write:users` for modifying user details.
 - `read:roles` or `write:roles` for managing roles.
 
-You can select scopes when you authorize a new application or you can add scopes to an existing application. We recommend adding as few scopes as you need, to maintain API security. 
+You can select scopes when you authorize a new application or you can add scopes to an existing application. We recommend adding as few scopes as you need, to maintain API security.
 
-## Change or add scopes to an application accessing the Kinde Management API
+## Request scopes in the body of an access token request
+
+By default token requests for an authorized application will return all the scopes enabled below. You can also optionally ask for a subset of enabled scopes to be returned by including them in the body of the access token request. You might do this to add more security to access requests, or because you want your users to be very specific about scopes in their requests.
```
Comment (DaveOrDead) [inline src/content/docs/developer-tools/kinde-api/about-m2m-scopes.mdx:29]: @clairekinde11 for management API it is unlikely their end users would be requesting the token. So probably
> You might do this to generate a token with limited access in order to minimize risk if the token was compromised

File: src/content/docs/developer-tools/kinde-api/about-m2m-scopes.mdx [hunk 2]
```diff
@@ -14,22 +14,39 @@ app_context:
...
+
+## Add or modify scopes for accessing the Kinde Management API
 
 Follow this procedure if you already have an application and you experience a scope error, if you want to add scopes for an application, or remove scopes to tighten security.
 
 1. Go to **Settings > Applications** and select the relevant M2M application.
 2. On the left, select **APIs**.
 3. Select the three dots next to the Kinde management API, then choose **Manage scopes**. 
 4. Select the scopes you want to include in the token. For maximum security only enable the minimum scopes you require. 
-5. Select **Save**. The scopes will now be included in the token. You do not need to also send them in the token request.
```
Comment (DaveOrDead) [inline src/content/docs/developer-tools/kinde-api/about-m2m-scopes.mdx:35]: @clairekinde11 if we wanted to keep something similar we could say:
> The scopes are now available to be included in the token

File: src/content/docs/developer-tools/your-apis/custom-api-scopes.mdx [hunk 3]
```diff
@@ -68,3 +68,20 @@ Take care deleting scopes. If a scope is in use, it can cause breaking changes f
 4. Select the dots menu (far right) and select:
    - **Edit scope.** You can only change the scope description. Select **Save**.
    - **Delete scope**. Confirm that you want to delete and select **Delete scope**.
+
+## Request a subset of scopes for an authorized application
+
+By default token requests for an authorized application will return all the scopes enabled in the section above. However, you can also optionally ask for a subset of enabled scopes to be returned by including them in the body of the access token request. You might do this to add more security to access requests for your API, or because you want your users to be very specific in their requests.
```
Comment (DaveOrDead) [inline src/content/docs/developer-tools/your-apis/custom-api-scopes.mdx:74]: Could include something like: 
> The fewer scopes your M2M access token has, the smaller the window of risk if the token was to be compromised.

Don't have to, just thinking out loud

File: src/content/docs/developer-tools/your-apis/test-token-from-kinde.mdx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 page_id: be39d3bb-a83d-4d21-8941-8f1c8363c3bb
-title: Get a user access token to test your APIs (Fast method)
+title: Get a M2M token to test your APIs
```
Comment (DaveOrDead) [inline src/content/docs/developer-tools/your-apis/test-token-from-kinde.mdx:3]: @clairekinde11 in my head it feels like `an` not `a` when I read it out loud `Get an M2M token to test your APIs`

Comment (clairekinde11) [reply src/content/docs/developer-tools/your-apis/test-token-from-kinde.mdx:3]: I know but ...

**Review by marcosmartini — APPROVED** [review]

**Pull request merged**